ios/Runner/bridges/FlutterBridgeSetup.swift

```diff
@@ -18,11 +20,16 @@ class FlutterBridgeSetup {
     }
     
     static func createBackgroundBridges(binaryMessenger: FlutterBinaryMessenger) {
-        
+        let backgroundSetupBridge = BackgroundSetupFlutterBridge()
+
+        backgroundSetupBridge.waitForBackgroundHandle().done {
+            FlutterBackgroundController.shared.setupEngine($0)
+        }.cauterize()
+
+        BackgroundSetupControlSetup(binaryMessenger, backgroundSetupBridge)
     }
     
     static func createUIBridges(binaryMessenger: FlutterBinaryMessenger) {
-        BackgroundSetupControlSetup(binaryMessenger, BackgroundSetupFlutterBridge())
```

Comment:
`createBackgroundBridges` is for all the (unimplemented currently) bridges which run on the background flutter isolate, since background setup runs before this its considered a UI bridge even if its kind of a misnomer so it still should be put in here (or in its own method maybe) as otherwise we end up initialising it again for the background code when its not needed.
tldr please move it back into `createUIBridges` (i get the confusion though!) and don't use `createBackgroundBridges` with the main flutter engine [here](https://github.com/pebble-dev/mobile-app/pull/204/files#diff-2d8ca8a3da302654c934d7f3be4c2f6c8edc7788f2b8ed0178730d6d703ebe72R29)

Reply:
Alrighty, I (kind of) understood what you are saying and pushed a suggestion, having `backgroundSetupBridge` as a private var. 
It is confusing with the names, I tried to give it some more structure with the 2 code marks. Tell me what you think - I am looking forward to get this in. To you it may have been a small thing but for me it was a great entry point to get into the project. Thanks for all the explanations 🙂